javascript/nanoexpress/config.yaml

```diff
@@ -0,0 +1,12 @@
+framework:
+  github: nanoexpress/nanoexpress
+  version: 2.4.4
```

Comment:
Could you use `X.Y` format ?

Reply:
Like `2.x.y`?

Reply:
like `2.4` because 
+`~2.4.4` in `package.json` could setup `2.4.8`
+ `2.4` is clearer for this project audience (I think)

Reply:
I fixed it, thanks